hathor/transaction/resources/mining.py

```diff
@@ -46,6 +46,9 @@ def render_GET(self, request):
         request.setHeader(b'content-type', b'application/json; charset=utf-8')
         set_cors(request, 'GET')
 
+        block_templates = self.manager.get_block_templates()
+        return json_dumpb([b.to_dict() for b in block_templates])
+
```

Comment:
No use for the code below here, right?

Reply:
Oh, I was using this for testing. And forgot to remove. It seems bad that mypy or flake8 didn't catch it.

Reply:
Removed.